glotaran/builtin/io/pandas/test/test_pandas_parameters.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+from glotaran.io import load_parameters
+from glotaran.io import save_parameters
+
+PATH_XLSX = Path.cwd().joinpath("glotaran/builtin/io/pandas/test/data/parameter.xlsx")
+PATH_YAML = Path.cwd().joinpath("glotaran/builtin/io/pandas/test/data/parameter.yaml")
+DATA_PATH = Path("testparameters.xlsx")
```

Comment:
This is highly dependent on where you execute pytest, better use the location of the current file.
```suggestion
PANDAS_TEST_DATA = Path(__file__).parent / "data"
PATH_XLSX = PANDAS_TEST_DATA  / "parameter.xlsx"
PATH_YAML = PANDAS_TEST_DATA  / "parameter.yaml"
```

Since `DATA_PATH` is only used in `test_save_parameters_xlsx`, it can be omitted.